x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/response_actions_log.tsx

```diff
@@ -597,7 +597,7 @@ export const ResponseActionsLog = memo<
   // create range label to display
   const recordRangeLabel = useMemo(
     () => (
-      <EuiText color="subdued" size="xs" data-test-subj={getTestId('endpointListTableTotal')}>
+      <EuiText color="default" size="xs" data-test-subj={getTestId('endpointListTableTotal')}>
```

Comment:
So according to the docs, `EuiText` should set the `default` color by default if no color prop is given. But when I tried that it translates to `color: inherit` instead of `color: #343741`. That's not what I expected, which is why I've explicitly given a color prop here. Also to avoid unintended "color bleed" from the parent element color in later changes.